src/utils/category.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Filter json array based on category prop
+ * @param {array} products - Array of product objects
```

Comment:
Writing a gist of what the file does and returns is a great way of providing a summary.